app/models/hubstats/deploy.rb

```diff
@@ -20,7 +20,15 @@ def check_time
     scope :belonging_to_users, lambda {|user_id| where(user_id: user_id.split(',')) if user_id}
     scope :with_repo_name, select('DISTINCT hubstats_repos.name as repo_name, hubstats_deploys.*').joins("LEFT JOIN hubstats_repos ON hubstats_repos.id = hubstats_deploys.repo_id")
     scope :with_user_name, select('DISTINCT hubstats_users.login as user_name, hubstats_deploys.*').joins("LEFT JOIN hubstats_users ON hubstats_users.id = hubstats_deploys.user_id")
-
+    
+    scope :belonging_to_teams, lambda {|teams|
+      if teams
```

Comment:
Add an acceptable return value if teams isn't provided or remove the conditional to make it required.